spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -15,10 +15,31 @@ we go.
 
 You need Java installed (Java 7 or better, we recommend Java 8), and to build, you need to have Maven installed as well.
 
-You also need to have link:http://redis.io/[Redis] installed and running if you plan on running the local server, or to run the included tests.  
+You need to have an RDBMS for storing stream, task and app states in the database. The `local` dataflow server by default uses embedded H2 database for this.
+
+You also need to have link:http://redis.io[Redis] running if you are running any streams that involve analytics applications. Redis may also be required run the unit/integration tests.
```

Comment:
change `may also be required run the unit/integration tests.`
 -to-
`is required to run the full suite of unit/integration tests.`